Gemfile

```diff
@@ -11,7 +11,7 @@ gem "rest-client"
 gem 'rake'
 gem 'redis'
 gem 'mixlib-versioning', '~> 1.1.0'
-gem 'mixlib-install'
+gem 'mixlib-install', '>= 3.11.23'
```

Comment:
Why do we need to pin mixlib-install here ?  Latest version of mixlib-install is "3.12.1"  which is already pulled in last update

Reply:
See: https://github.com/chef/omnitruck/pull/420#discussion_r357954759